libs/version/readme.md

```diff
@@ -38,21 +38,44 @@ int main(int argc, char *argv[])
 
 ## 2. Usage in Makefile
 
-At the end of your makefile add the following line:
+At the end of your makefile, you need to include
+```$(BLOCKSDS)/libs/version/blocksds_version.make```.
+Below, an example which includes the file and also checks
+for older versions of the BlocksDS SDK as well.
+Depending on the use case, checking for the existence
+of ```$(BLOCKSDS)/libs/version/blocksds_version.make``` may not be necessary.
 
 ```make
-include $(BLOCKSDS)/libs/version/blocksds_version.make
+BLOCKSDS_VERSION_FILE = $(BLOCKSDS)/libs/version/blocksds_version.make
+
+ifeq ("$(wildcard $(BLOCKSDS_VERSION_FILE))","")
+$(error BlocksDS version too old. Please update.)
+endif
+
+include $(BLOCKSDS_VERSION_FILE)
 ```
 
-After that line you can use it like this, for example:
+After those lines you can use it like this, for example:
 
 ```make
+# This will return 1 if the installed version of BlocksDS is v1.14.2.
+# It will return 0 otherwise.
+$(call blocksds_version_equals, 1, 14, 2)
```

Comment:
This statement by itself (and the ones below) are useless. I would show how to assign this to a variable.